sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -118,7 +118,7 @@ public class DefaultWizard implements Wizard {
 
 	public static final String DB_OPTION_MYSQL = "MySQL 5.6 and above (requires pre-installed MySQL 5.6 and above)";
 
-	public static final String DB_OPTION_SDK_DOCKER_MYSQL = "MySQL 5.6 and above in SDK docker container (requires pre-installed Docker)";
+	public static final String DB_OPTION_SDK_DOCKER_MYSQL = "MySQL 8.4.1 and above in SDK docker container (requires pre-installed Docker)";
```

Comment:
Do we need `and above`?

Reply:
"I don't think it's necessary to specify a version there at all. I made the change because we had previously specified which version of MySQL docker uses."